common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,63 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 getVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist getTwist(const geometry_msgs::msg::Vector3 & velocity)
+{
+  geometry_msgs::msg::Twist twist;
+  twist.linear = velocity;
+  return twist;
+}
+
+inline geometry_msgs::msg::TwistWithCovariance getTwistWithCovariance(
+  const geometry_msgs::msg::Twist & twist)
+{
+  geometry_msgs::msg::TwistWithCovariance twist_with_covariance;
+  twist_with_covariance.twist = twist;
+  return twist_with_covariance;
+}
+
+inline geometry_msgs::msg::TwistWithCovariance getTwistWithCovariance(
+  const geometry_msgs::msg::Vector3 & velocity)
+{
+  geometry_msgs::msg::TwistWithCovariance twist_with_covariance;
+  twist_with_covariance.twist = getTwist(velocity);
+  return twist_with_covariance;
+}
+
+inline double getTwistNorm(const geometry_msgs::msg::Vector3 & v)
+{
+  return std::sqrt(v.x * v.x + v.y * v.y + v.z * v.z);
+}
+
+inline double getTwistNorm(const geometry_msgs::msg::Twist & twist)
+{
+  return getTwistNorm(twist.linear);
+}
+
+inline double getTwistNorm(const geometry_msgs::msg::TwistWithCovariance & twist_with_covariance)
+{
+  return getTwistNorm(twist_with_covariance.twist.linear);
+}
+
+// Judge whether twist covariance is available.
+// If all element of covariance is 0, return false.
+inline bool isTwistCovarianceAvailable(
```

Comment:
Where is this function used?
What's the definition of "available"? Did you mean "valid"?

Reply:
You are right, and sorry for your confusing.
Fixed at [909f88b](https://github.com/autowarefoundation/autoware.universe/pull/2049/commits/909f88b6a0c2b2a0b860f6aee2a8243e8ea3b7d2)

Reply:
Could you show me the example use case?

Reply:
Use case is here

Radar fusion
https://github.com/autowarefoundation/autoware.universe/blob/main/perception/radar_fusion_to_detected_object/src/radar_fusion_to_detected_object.cpp#L133
https://github.com/autowarefoundation/autoware.universe/blob/main/perception/radar_fusion_to_detected_object/src/radar_fusion_to_detected_object.cpp#L111

Reply:
Okay, I understand it's for https://github.com/tier4/autoware_auto_msgs/blob/bd162589a837a2c1ecea943f80c035109e1ae594/autoware_auto_perception_msgs/msg/DetectedObjectKinematics.idl#L38.